tests/cd_command_test.sh

```diff
@@ -1,4 +1,4 @@
-set -uo pipefail
+set -u
```

Comment:
We would need a more recent version of Zsh to support `pipefail` in CI.

Reply:
you mean then the one provided by travis? can we install it or it's not really available for the version of ubuntu they are providing?

Reply:
Yes, the one provided by the version of ubuntu available on Travis.